core/storage/exploration/gae_models.py

```diff
@@ -239,6 +239,53 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
+class MathExplorationImagesModel(base_models.BaseModel):
```

Comment:
ExplorationMathExpressionStatsModel (or something around that!)

Reply:
done